keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -543,6 +550,39 @@ Predefined condition types:
   The status of the cluster SHOULD be updated by the cluster manager under
   this condition.
 
+### Cluster Access
+There are multiple methods for a ClusterInventory Consumer to gain access to the cluster represented by a ClusterProfile API.
+This KEP does not define the exact mechanism for each approach, but it is recommended that ClusterInventory Consumers avoid 
+using a secret if possible. Here are the three approaches:
+
+#### Pull Model
+The ClusterInventory Consumer can leverage the [Work API](https://multicluster.sigs.k8s.io/concepts/work-api/) so that
+the work API agent on the Member Cluster can *pull* the necessary objects from the management cluster where the ClusterInventory 
+Consumer operates to the Member Cluster. This approach allows the Cluster Manager to manage the RBAC for each Consumer on the 
+management cluster, ensuring that their access is restricted to the corresponding namespaces of the ClusterProfile objects.
+
+#### Identity Federation 
+The ClusterInventory consumer can utilize identity federation mechanisms, such as [Azure Workload Identity Federation](https://learn.microsoft.com/en-us/entra/workload-id/workload-identity-federation)
+or [GCP Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation). This allows the 
+ClusterInventory Consumer to use an identity that already has access to the clusters in the Cluster Inventory.
+While the Cluster Manager can assist in setting up the federation, it is not a mandatory requirement.
+
+#### Access via Secret
+The ClusterInventory Consumer can obtain credentials to access the cluster represented by a ClusterProfile object by reading 
+from a secret. In this approach, the Cluster Manager generates secrets containing the necessary credentials within the namespace
+accessible to the ClusterInventory Consumer. For this to function correctly, Cluster Managers must be aware of the following details 
+about the consumer: their name, whether credentials are required, and the preferred unique namespace for reading credentials as secrets.
+Those information can be obtained during the "registering" process but this is out of the scope of this KEP.
```

Comment:
Should we add that it is necessary that the "ClusterManager properly manipulates the credentials and offers rotation and expiration control as necessary depending on the type of credentials" ?

Or something to put burden on the ClusterManager to make sure the credentials are somewhat properly handled?

Reply:
I haven’t checked whether other KEPs mentioning credentials say similar things; but if we do add something along those lines, I think it should go in the introductory section in “Cluster Access” because it applies across the board (even when it’s effectively delegated to cloud providers etc.).